app/services/pii/attributes.rb

```diff
@@ -5,6 +5,7 @@ module Pii
 
   Attributes = RedactedStruct.new(
     :first_name, :middle_name, :last_name,
+    :state_id_address1, :state_id_address2, :state_id_city, :state_id_zipcode,
```

Comment:
should we include `state_id_jurisdiction`?

Reply:
Really good point, thanks!  @sheldon-b is looking into it [here](https://github.com/18F/identity-idp/pull/7961#discussion_r1135700194).  (I'm not going to make a change here since he is looking into it.  If there is a merge conflict at any point, I'll resolve it in favor of his changes.)